src/RA_PopupWindows.h

```diff
@@ -41,11 +42,11 @@ class PopupWindows
 };
 
 //	Exposed to DLL
-extern "C"
-{
-    API extern int _RA_UpdatePopups(ControllerInput* input, float fDTime, bool Full_Screen, bool Paused);
-    API extern int _RA_RenderPopups(HDC hDC, RECT* rcSize);
-}
+_EXTERN_C
+_NODISCARD API int _RA_UpdatePopups(_In_ const ControllerInput* const __restrict input, _In_ float fDTime,
+                                    _In_ bool Full_Screen, _In_ bool Paused);
+_NODISCARD API int _RA_RenderPopups(_In_ HDC hDC, _In_ const RECT* const __restrict rcSize);
+_END_EXTERN_C
```

Comment:
These cannot be `_NODISCARD`. The calling methods are currently ignoring the return value: https://github.com/RetroAchievements/RAIntegration/blob/master/src/RA_Interface.cpp#L114-L118

Reply:
Alright. Just for future reference you can use (void) to suppress a nodiscard warning.

Reply:
Just to note, we don't actually need gsl to use the suppression because only CppCoreCheck understands it.